tests/js/spec/components/assigneeSelector.spec.jsx

```diff
@@ -136,7 +136,7 @@ describe('AssigneeSelector', function() {
 
   it('should initially have loading state', function() {
     openMenu();
-    expect(assigneeSelector.find('LoadingIndicator')).toHaveLength(1);
+    expect(assigneeSelector.find('LoadingIndicator')).toHaveLength(2);
```

Comment:
are you sure to change this to `2` I believe we should not get 2 LoadingIndicator nodes.